### DISCUSSION TITLE
XHS-Spider

### DISCUSSION THREAD

**Discussion by xisuo67** [xisuo67/XHS-Spider#7] [Announcements]
## 👋 欢迎！
  我们正在使用讨论作为与社区其他成员联系的地方。我们希望您：
  * 问你想知道的问题。
  * 分享意见。
  * 与其他社区成员互动。
  * 欢迎他人，思想开放。请记住，这是一个我们共同建立的社区💪.
  首先，请在下方发表评论并介绍您自己，并告诉我们您对这个社区所做的工作。

**Comment by W2426566602** [comment]
嘿嘿 可以可以可以

**Comment by W2426566602** [comment]
66666666666666666

**Comment by nikpictures** [comment]
安装了NET 6.0还是闪退

**Reply by xisuo67** [reply]
https://github.com/xisuo67/XHS-Spider/issues/32
